src/quo2/foundations/elevation.cljs

```diff
@@ -0,0 +1,70 @@
+(ns quo2.foundations.elevation
+  (:require [quo2.foundations.colors :as colors]
+            [quo2.theme :as theme]))
+
+
+(defn- get-inverted [inverted? number] (if inverted? (* -1 number) number))
+
+(defn- get-scales
+  [inverted?]
+  (if (theme/dark?)
+    {:elevation-1 {:shadow-color   (colors/alpha colors/neutral-100 0.5)
+                   :shadow-offset  {:width  0
+                                    :height (get-inverted inverted? 4)}
+                   :elevation      3
```

Comment:
could you please elaborate on why it starts from 3 , and also i believe elevation is relative, so if it will be in the container with different elevation there will be different shadow ?

Reply:
tbh, I am not very familiar with elevation. I will read up more on it. My intention here was to get this setup and then work with a designer to configure the values directly. The values in figma are for css and it's not very clear of the 1-1 mapping between react-native shadows and css shadows. especially since ios and android are handled separately. In any case we can add the correct named shadows into the codebase and then devs can start using these named shadows as in the designs and will get the correct updates once I'm working with the design team.

Reply:
great, thanks